kubernetes-extension/src/main/ballerina/ballerinax/kubernetes/annotation.bal

```diff
@@ -93,6 +93,20 @@ public type ConfigMapKeyRef record {
     !...
 };
 
+@final public ImagePullPolicy IF_NOT_PRESENT = "IfNotPresent";
```

Comment:
We cannot use these variables as of yet as they are not values during compile time. Have to wait for compile time constants.